nextflow_schema.json

```
@@ -148,19 +148,23 @@
                     "pattern": "^\\S+\\.csv$",
                     "description": "Path to comma-separated file containing information about the samples in the experiment.",
                     "help_text": "You will need to create a design file with information about the samples in your experiment before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row. See [usage docs](https://nf-co.re/proteinfold/usage#samplesheet-input).",
-                    "fa_icon": "fas fa-file-csv"
+                    "fa_icon": "fas fa-file-csv",
+                    "errorMessage": "Input samplesheet must be a CSV file that exists. Please check the file path and format."
                 },
                 "outdir": {
                     "type": "string",
                     "format": "directory-path",
                     "description": "The output directory where the results will be saved. You have to use absolute paths to storage on Cloud infrastructure.",
-                    "fa_icon": "fas fa-folder-open"
+                    "fa_icon": "fas fa-folder-open",
+                    "errorMessage": "Output directory path must be specified"
                 },
                 "mode": {
                     "type": "string",
                     "default": "alphafold2",
                     "description": "Specifies the mode in which the pipeline will be run. mode can be any combination of ['alphafold2', 'colabfold', 'esmfold'] separated by a comma (',') with no spaces.",
-                    "fa_icon": "fas fa-cogs"
+                    "fa_icon": "fas fa-cogs",
+                    "pattern": "^(alphafold2|colabfold|esmfold)(,(alphafold2|colabfold|esmfold))*$",
+                    "errorMessage": "Mode must be a comma-separated list containing any combination of 'alphafold2', 'colabfold', or 'esmfold'"
```

Adding the two new modules available on the dev branch:
```suggestion
                    "pattern": "^(alphafold2|colabfold|esmfold|rosettafold_all_atom|helixfold3)(,(alphafold2|colabfold|esmfold|rosettafold_all_atom|helixfold3))*$",
                    "errorMessage": "Mode must be a comma-separated list containing any combination of 'alphafold2', 'colabfold', 'esmfold', 'rosettafold_all_atom', or 'helixfold3'"
```


Future modules not yet pulled to dev:
```suggestion
                    "pattern": "^(alphafold2|colabfold|esmfold|rosettafold_all_atom|helixfold3|boltz|chai1)(,(alphafold2|colabfold|esmfold|rosettafold_all_atom|helixfold3|boltz|chai1))*$",
                    "errorMessage": "Mode must be a comma-separated list containing any combination of 'alphafold2', 'colabfold', 'esmfold', 'rosettafold_all_atom', 'helixfold3', 'boltz', or 'chai1'"
```